### DISCUSSION TITLE
db export

### DISCUSSION THREAD

**Discussion by g4o3** [g4o2/PHP-SQL-Chat#4] [Q&A] [answered]
## Chat database export
Can I have the db export file of your chat?

**Comment by Maxhu787** [comment, marked as answer]
[g4o2-chat-2023-1-13.txt](https://github.com/g4o2/PHP-SQL-Chat/files/10465871/g4o2-chat-2023-1-13.txt)

**Comment by Maxhu787** [comment]
[g4o2-chat-2023-5-27.zip](https://github.com/g4o2/PHP-SQL-Chat/files/11645759/g4o2-chat-2023-5-27.zip)
![database diagram](https://github.com/g4o2/PHP-SQL-Chat/assets/103299803/259b126b-adfb-43a5-8f4d-af49a9d1c473)
